web/src/containers/Withdraw/index.tsx

```diff
@@ -76,8 +76,8 @@ class WithdrawComponent extends React.Component<Props, WithdrawState> {
     public componentWillReceiveProps(nextProps) {
         const { currency, withdrawDone } = this.props;
 
-        if ((nextProps && (JSON.stringify(nextProps.currency) !== JSON.stringify(currency))) || (nextProps.withdrawDone && !withdrawDone)) {
-            this.clearFields();
+        if ((nextProps && (nextProps.currency !== currency) || nextProps.withdrawDone && !withdrawDone)) {
```

Comment:
The bracket grouping has changed from the original one. It should be like below ?
if ((nextProps && (nextProps.currency !== currency)) || (nextProps.withdrawDone && !withdrawDone)) {